app/components/Base/index.js

```diff
@@ -10,41 +10,41 @@ import {
   BORDER_PREMIUM_RGB,
   BORDER_RADIUS_L,
   BORDER_TUTORIAL,
-} from 'style-constants';
+} from "style-constants";
 
 const borderTopLeftRadius = ({ bordered, topRightRadius, withoutBR }) =>
-  (bordered || topRightRadius) && !withoutBR ? BORDER_RADIUS_L : 'none';
+  (bordered || topRightRadius) && !withoutBR ? BORDER_RADIUS_L : "none";
 
 const borderTopRightRadius = ({ bordered, bottomRightRadius, withoutBR }) =>
-  (bordered || bottomRightRadius) && !withoutBR ? BORDER_RADIUS_L : 'none';
+  (bordered || bottomRightRadius) && !withoutBR ? BORDER_RADIUS_L : "none";
 
 const Base = styled.div`
   background: ${BG_LIGHT};
   padding: 20px 30px;
   flex-grow: 1;
   padding-top: ${({ paddingTop }) => paddingTop || 20}px;
 
-  overflow: ${({ overflowHidden }) => (overflowHidden ? 'hidden' : 'initial')};
+  overflow: ${({ overflowHidden }) => (overflowHidden ? "hidden" : "initial")};
   border: ${({ bordered, isPromoted, isTutorial }) =>
     bordered || isPromoted
       ? `1px solid ${
           isPromoted
             ? BORDER_PREMIUM
             : isTutorial
```

Comment:
second ternar should be extracted to own function

Reply:
The second ternar is not my solution. I had some errors When I tried to refactor this whith creating  own function (